Gruntfile.js

```diff
@@ -137,7 +137,7 @@ module.exports = function( grunt ) {
 						"ascii_only": true
 					},
 					banner: "/*! jQuery v<%= pkg.version %> | " +
-						"(c) 2005, <%= grunt.template.today('yyyy') %> jQuery Foundation, Inc. | " +
+						"Copyright jQuery Foundation and other contributors " +
```

Comment:
Why the change from `(c)` to `Copyright`? You didn't do the same in the Sizzle PR: https://github.com/jquery/sizzle/pull/307/files

This is the banner put in the minified files, e.g. https://code.jquery.com/jquery-2.1.3.min.js